spk/squidguard/src/installer.sh

```diff
@@ -37,28 +37,40 @@ postinst ()
     # Patch template files
     hostname=`hostname`
     sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidguard.conf.tpl > ${ETC_DIR}/squidguard.conf
-    
+    sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidclamav.conf.tpl > ${ETC_DIR}/squidclamav.conf    
     # Correct the files ownership
     chown -R ${RUNAS}:users ${SYNOPKG_PKGDEST}
-    chown 0:0 ${SQUID_WRAPPER}
-	chmod +s ${SQUID_WRAPPER}
-	
-    # Init squid cache directory
-    su - ${RUNAS} -c "${SQUID} -z -f ${CFG_FILE}"
+
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        # Init squid cache directory
+        su - ${RUNAS} -c "${SQUID} -z -f ${CFG_FILE}"
+    fi
+
+    # Init SSLBump cache directory
+    su - ${RUNAS} -c "${INSTALL_DIR}/libexec/ssl_crtd -c -s ${INSTALL_DIR}/var/ssl_db" >> /dev/null
 
     # Install webman
```

Comment:
This might not have been implemented at the time this package was originally created, but can you check out the use of `DSM_UI_DIR = app` in the various packages and Developers Guide. Using that will probably clean up a few different parts of code in this file and several others.